Lesson6/Lesson6/Figure.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lesson6
+{
+    abstract public class Figure
```

Comment:
Модификаторы в неправильном (нестандартном) порядке.
```suggestion
    public abstract class Figure
```